metaflow/plugins/kubernetes/kubernetes_job.py

```diff
@@ -563,9 +564,24 @@ def has_failed(self):
     @property
     def is_running(self):
         # Returns true if the pod is running.
-        return not self.is_done and (
-            self._pod.get("status", {}).get("phase") == "Running"
-        )
+        if self.is_done:
+            return False
+
+        container_statuses = self._pod.get("status", {}).get("container_statuses", [])
+        if not container_statuses:
+            # Yet to start, most likely
+            return True
+        if len(container_statuses) < 1:
+            # Yet to start, most likely
+            return True
```

Comment:
should we return from pod phase here and above?

Reply:
Done